### ISSUE TITLE
New Term - Brainfu*k

### ISSUE THREAD

**Issue by sasakiRoo** [Buzzpy/Dev-Encyclopedia#172]
![Issue Banner Image](https://github.com/user-attachments/assets/272cd422-01a7-41d3-a09f-b13e2ad004ae)

Following is/are the terms I'd like to add: 
- Term: Brainfu*ck 
- Subtext: An Esoteric Programming Langugue
- Explanation: Term explanation, in plain English
- Resource URL: https://en.wikipedia.org/wiki/Brainfuck
- Image URL (Optional, but recommended): URL of an image related to the term

**Regarding the PR for this issue,**
- [ ] I won't create the PR, **assign someone else** for this

**Comment by phunlh2001** [comment]
Do you have any images that represent this language?

**Comment by sasakiRoo** [comment]
here you go!
![brainfuck_code](https://github.com/user-attachments/assets/8178b257-faa3-435d-8278-b0b979787adf)

**Comment by Buzzpy** [comment]
Closing the issue as we added the term! 🚀
